### PR TITLE
Skip abstract definitions in compiler passes

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #21761
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Review by nicolas-grekas — APPROVED** [review]
:+1:

File: src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPassTest.php [hunk 1]
```diff
@@ -32,12 +29,10 @@ public function testExceptionOnAbstractTaggedSubscriber()
         $container->setDefinition('a', $abstractDefinition);
 
         $this->process($container);
+        $this->assertSame(array(), $container->getDefinition('doctrine.dbal.default_connection.event_manager')->getMethodCalls());
```
Comment (GuilhemN) [inline src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPassTest.php]: `assertEmpty()`?

**Review by GuilhemN — APPROVED** [review]
:+1:

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**